packages/_private/react-demo/src/Layout.tsx

```diff
@@ -22,7 +22,15 @@ export const Layout: FC = () => {
 
   return (
     <>
-      <SynHeader label="@synergy-design-system/react Form Demo">
+      <SynHeader label="@synergy-design-system/react Components Demo">
+        <Link
+          className="custom-logo"
+          slot="logo"
+          to="/"
+        >
+          <SynIcon name="logo-color" library="system" />
+        </Link>
```

Comment:
This (using an <a/> tag around a syn-icon in the logo slot) makes the header look wrong with safari. There is too much space at the left side of the logo. See image in angular demo

Reply:
Fixed. We did not get this initially as Storybook was not tested with Safari when we created the header.